trace/config.json

```
@@ -26,10 +26,43 @@
         "#005900", "#000078", "#490D00",
         "#8A034F", "#005A8A", "#443500"
     ],
-    "colors": [
+    "default": [
```

Related to other suggestion here:
https://github.com/slaclab/trace/pull/199/files#r2350178188

You can make the palettes more dynamic by changing the config file to better reflect the structure of the the `color_palette` variable we want to use. Here is one way we can do it that matches with the comment referenced above:


```suggestion
    "color_palettes": {
        "default": [
            "#008CF9", "#006E00", "#B80058",
            "#EBAC23", "#B24502", "#FF9287",
            "#5954D6", "#00C6F8", "#878500",
            "#00A76C", "#D163E6", "#00BBAD"
        ],
        "java": [
            "#FF0000", "#0000FF", "#00FF00",
            "#FF00FF", "#000000", "#660099",
            "#FF6600", "#339999", "#999999",
    ...
```